tests/spread/integration/dotnet-runtime-9.0/task.yaml

```diff
@@ -0,0 +1,21 @@
+summary: Integration tests for .NET 9 Runtime 
+
+execute: |
+  # install slices
+  rootfs="$(install-slices dotnet-runtime-9.0_libs base-files_base)"
+  mv "${rootfs}"/usr/lib/dotnet/dotnet9 "${rootfs}"/usr/lib/dotnet/dotnet
+  ln -s ../lib/dotnet/dotnet "${rootfs}"/usr/bin/dotnet
+
+  # smoking test the .NET runtime
+  chroot "${rootfs}" /usr/bin/dotnet --info
```

Comment:
We can remove this. See `tests/spread/integration/aspnetcore-runtime-9.0/task.yaml`